workspace/src/app/views/shared/FileUploader/cases/UploadNewFile/RetryFileItem.tsx

```diff
@@ -19,6 +19,7 @@ export function RetryFileItem({ file, onCancelRetry, onRetry }: IProps) {
             <Notification
                 onDismiss={() => onCancelRetry(file.id)}
                 danger={true}
+                timeout={1000 * 60 * 60}
```

Comment:
As this is workaround for a bug in a sub library please mark it with `@workaround` and add a short TODO or FIXME comment about it and a link to https://chat.eccenca.com/eccenca/pl/mqp79pyr1jy5un1tkstbkomf5e because we need to change it back later.